app/main.py

```diff
@@ -1,8 +1,32 @@
+from typing import List, Dict, Any
+
+
 class Person:
-    # write your code here
-    pass
+    people: Dict[str, "Person"] = {}
```

Comment:
Em vez de usar o typing Dict, você pode utilizar o dict direto, dessa forma

```suggestion
    people: dict[str, "Person"] = {}
```